burstphoto/ContentView.swift

```diff
@@ -6,9 +6,9 @@ enum AppState {
 }
 
 class AppSettings: ObservableObject {
-    @AppStorage("tile_size") static var tile_size: Int = 16
-    @AppStorage("search_distance") static var search_distance: String = "Medium"
```

Comment:
I would strongly urge you to not remove the search distance. I found that for schemes containing repeating patterns (e.g. bricks) that it's better to reduce the search distance as otherwise there is a tendency to merge with wrong portions of the pattern, and doing so in ways that it visible as an artifact.